assets/src/edit-story/components/panels/panel/useKeyboardHandlers.js

```diff
@@ -18,34 +18,29 @@
  * External dependencies
  */
 import { useCallback, useLayoutEffect } from 'react';
+import Mousetrap from 'mousetrap';
```

Comment:
To clarify, i'm not entirely sure what's this module is otherwise doing. But can we use `useKeyDownEffect` and family or is this use cases sufficiently different in some way?

Reply:
+1, useKeyEffect can be used instead.
Done.